backend/clap/proto/data_store.proto

```diff
@@ -111,9 +114,23 @@ message RatingsSubmissionResponse {
     AssessorRatings final      = 4;
 }
 
-service DataStore {
-    rpc GetSubmissions      (SubmissionRequest)        returns (SubmissionsResponse)       {}
-    rpc GetSubmissionDetails(DetailsSubmissionRequest) returns (DetailsSubmissionResponse) {}
-    rpc GetSubmissionRatings(RatingsSubmissionRequest) returns (RatingsSubmissionResponse) {}
+message NewSubmissionRatingRequest {
+    RatingType type = 1;
 }
 
+message PartialRatingRequest {
+    int32 criterion_id      = 1;
+    int32 rating_id         = 2;
+    int32 partial_rating_id = 3;
+    string justification    = 4;
+    int32 points            = 5;
+}
+
+service DataStore {
+    rpc GetSubmissions          (SubmissionRequest)           returns (SubmissionsResponse)       {}
+    rpc GetSubmissionDetails    (DetailsSubmissionRequest)    returns (DetailsSubmissionResponse) {}
+    rpc GetSubmissionRatings    (RatingsSubmissionRequest)    returns (RatingsSubmissionResponse) {}
+    rpc PostNewSubmissionRating (NewSubmissionRatingRequest)  returns (Rating)                    {}
+    rpc PostPartialRating       (PartialRatingRequest)        returns (PartialRating)             {}
+    rpc PostSubmitRating        (google.protobuf.Empty)       returns (PartialRating)             {}
```

Comment:
so.. when datastore gets call on this endpoint what is it supposed to do... with no data, no id? When BM said that it requires no body it meant HTTP body. But clap gets `ratingId` from path variable which must me included as grpc request body